### ISSUE TITLE
Housing point to add presentations/talks.

### ISSUE THREAD

**Issue by koomie** [openhpc/ohpc#183]
* [OpenHPC-Overview-fosdem16.pdf](https://github.com/openhpc/ohpc/files/146753/OpenHPC-Overview-fosdem16.pdf)

**Comment by boegel** [comment]
also (link to) the video of your talk?

http://video.fosdem.org/2016/aw1126/openhpc-community-building-blocks-for-hpc-systems.mp4

**Comment by koomie** [comment]
Certainly! (although we should pity the soul for the individual who is willing to watch me ramble).

**Comment by koomie** [comment]
[OpenHPC-ISC-2016-BoF.pdf](https://github.com/openhpc/ohpc/files/358034/OpenHPC-ISC-2016-BoF.pdf)

**Comment by koomie** [comment]
[OpenHPC-overview-mug-final-2016.pdf](https://github.com/openhpc/ohpc/files/474943/OpenHPC-overview-mug-final-2016.pdf)

**Comment by koomie** [comment]
[SC16-BoF-OpenHPC-overview.pdf](https://github.com/openhpc/ohpc/files/596158/SC16-BoF-OpenHPC-overview.pdf)

**Comment by koomie** [comment]
[HPCSYSPROS16_OpenHPC.pdf](https://github.com/openhpc/ohpc/files/619162/HPCSYSPROS16_OpenHPC.pdf)

**Comment by crbaird** [comment]
[2017-OpenHPC-BoF-Session.pdf](https://github.com/openhpc/ohpc/files/1479095/2017-OpenHPC-BoF-Session.pdf)

**Comment by crbaird** [comment]
[2017-OpenHPC-BoF-Session.pdf](https://github.com/openhpc/ohpc/files/1479463/2017-OpenHPC-BoF-Session.pdf)

**Comment by koomie** [comment]
[openhpc-updates-mug-2018.pdf](https://github.com/openhpc/ohpc/files/2352990/openhpc-updates-mug-2018.pdf)

**Comment by adrianreber** [comment]
[2018-devconf.cz-openhpc-introduction.pdf](https://github.com/openhpc/ohpc/files/2416462/2018-devconf.cz-openhpc-introduction.pdf)
[2018-centos-dojo-brussels-openhpc-and-centos.pdf](https://github.com/openhpc/ohpc/files/2416496/2018-centos-dojo-brussels-openhpc-and-centos.pdf)
[2018-tuebix-openhpc.pdf](https://github.com/openhpc/ohpc/files/2416517/2018-tuebix-openhpc.pdf)
[2018-hpc-symposium-freiburg-openhpc.pdf](https://github.com/openhpc/ohpc/files/2416523/2018-hpc-symposium-freiburg-openhpc.pdf)

**Comment by koomie** [comment]
[OpenHPC-DAAC-2018.pdf](https://github.com/openhpc/ohpc/files/2589833/OpenHPC-DAAC-2018.pdf)

**Comment by koomie** [comment]
[2019-OpenHPC-BoF-Session.pdf](https://github.com/openhpc/ohpc/files/3874741/2019-OpenHPC-BoF-Session.pdf)

**Comment by koomie** [comment]
[2021-ISC-BoF.pdf](https://github.com/openhpc/ohpc/files/6733785/2021-ISC-BoF.pdf)

**Comment by koomie** [comment]
[ohpc-sc21-bof.pdf](https://github.com/openhpc/ohpc/files/7557447/ohpc-sc21-bof.pdf)

**Comment by ECoulter** [comment]
PEARC22 BoF Slides [ohpc-pearc22-bof.pptx](https://github.com/openhpc/ohpc/files/9199835/ohpc-pearc22-bof.pptx)
PEARC22 BoF Q&A [bif52kuv_-N3H3xpn4cX55EMDQYJm_contributions.csv](https://github.com/openhpc/ohpc/files/9199837/bif52kuv_-N3H3xpn4cX55EMDQYJm_contributions.csv)

**Comment by koomie** [comment]
[ohpc-pearc22-bof.pdf](https://github.com/openhpc/ohpc/files/9199858/ohpc-pearc22-bof.pdf)

**Comment by tardigradus** [comment]
I would be interested in seeing the slides for https://sc25.supercomputing.org/proceedings/bof/bof_pages/bof177.html, assuming there were any.

**Comment by adrianreber** [comment]
> I would be interested in seeing the slides for https://sc25.supercomputing.org/proceedings/bof/bof_pages/bof177.html, assuming there were any.

Let me see if I can find someone who still has it.
